Tests/Functional/Hooks/class.tx_multicolumn_tcemainTest.php

```diff
@@ -52,111 +55,167 @@ protected function setUp()
     }
 
     /**
+     * get the current UID from the LAST copied parent UID
+     *
+     * @param $oldUid
+     * @return int
+     */
+    protected function getUIDFromCopy($oldUid)
```

Comment:
This function is not needed. You can use $dataHandler->copyMappingArray[$table][$origUid]